ui/src/index.tsx

```diff
@@ -546,4 +546,6 @@ if (rootNode) {
   render(<Root />, rootNode)
 }
 
-writeNavigationTimingMetrics()
+window.addEventListener('load', _event => {
```

Comment:
```suggestion
window.addEventListener('load', () => {
```
generally you use the `_argument` syntax when you have a callback function and you need to use an argument that is later in the list _and_ you don't want to use the earlier argument:

```ts
doSomethingThenCallCallback((_event, metric) => {
  writeMetric(metric)
  // don't care about the first argument `event`, but need to list it
  // to access `metric` in the argument list
})
```

Reply:
I changed this!